src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1134,3 +1133,15 @@ def get_device_lvs(device, name_prefix=''):
     lvs = _output_parser(stdout, LV_FIELDS)
     return [Volume(**lv) for lv in lvs if lv['lv_name'] and
             lv['lv_name'].startswith(name_prefix)]
+
+def lvpath_to_lv(lv_path):
```

Comment:
nit: the parameter received is not exactly path, it would be nicer if we have a name more precise.

Reply:
updated

Reply:
Thanks!